oak-store-document/src/main/java/org/apache/jackrabbit/oak/plugins/document/UpdateOp.java

```diff
@@ -557,6 +590,16 @@ public static Condition newNotEqualsCondition(@Nullable Object value) {
             return new Condition(Type.NOTEQUALS, value);
         }
 
+        /**
+         * Creates a new lessThan condition with the given value.
+         *
+         * @param value the value to compare t
+         * @return the lessThan condition.
+         */
+        public static Condition newLessThanCondition(@Nullable Object value) {
```

Comment:
What is the semantic when `value` is `null`? I think it would be better if `value` is of type `Comparable<T>`, similar to `UpdateOp.max()`.

I'd also like to better understand if this addition is really necessary. Adding this functionality should also have more test coverage. E.g. additional tests in `BasicDocumentStoreTest`.

Reply:
I have removed all the logic related with "lessThan", as now I'm using UpdateOp.max() as you proposed.